scene/main/timer.cpp

```diff
@@ -211,6 +223,7 @@ void Timer::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "one_shot"), "set_one_shot", "is_one_shot");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "autostart"), "set_autostart", "has_autostart");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "paused", PROPERTY_HINT_NONE, "", 0), "set_paused", "is_paused");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "time_scale", PROPERTY_HINT_EXP_RANGE, "0.01, 4096, 0.01"), "set_time_scale", "get_time_scale");
```

Comment:
If a value of `0.0` is valid, we should allow it in the inspector. Also, the hint should allow more precision and use a lower maximum value for easier editing but allow manually entering greater values:

```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "time_scale", PROPERTY_HINT_EXP_RANGE, "0.0,100,0.001,or_greater"), "set_time_scale", "get_time_scale");
```